### PR TITLE
Clinic database based on a diagram.

### PR DESCRIPTION
**Pull request by franclobo**
### Clinic database based on a diagram.

- Create a file named schema_based_on_diagram.sql where we implement the database from the diagram.
- Create a join table (Medical_treatment) from many-to-many relationships.
- Add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    NOT VALID;
+    
+-- Join invoice_items with treatments.
+ALTER TABLE IF EXISTS public.invoice_items
+    ADD CONSTRAINT fk_treatment FOREIGN KEY (treatment_id)
+    REFERENCES public.treatments (id) MATCH SIMPLE
+    ON UPDATE NO ACTION
+    ON DELETE NO ACTION
+    NOT VALID;
+
```
Comment (danmainah) [inline schema_based_on_diagram.sql]: - [x] Kindly ensure all foreign keys have indexes created eg `CREATE INDEX ON medical_histories (patient_id);`

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @franclobo , @Yilikal250 and @AntonioHincapie , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (franclobo) [reply schema_based_on_diagram.sql]: Created indexes for all foreign keys.

**Review by DammyShittu — APPROVED** [review]
Hi team,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Creation of indexes

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**